lib/statsd/instrument/backends/udp_backend.rb

```diff
@@ -76,6 +77,15 @@ def generate_packet(metric)
       command
     end
 
+    def packet_tags(tags)
+      if self.tags_whitelist
+        tags = tags.select { |tag| self.tags_whitelist.any? { |white| tag.start_with?(white) } }
```

Comment:
I'd really prefer this to be `tags_whitelist.include?(something)` ion combination with using a set, which will be a lot faster, especially for a large whitelist.

Reply:
If we could do that we could even use `tags &= self.tags_whitelist` -- that was my original plan, before I realised that the useful tags are all dynamic things where we only know the prefix statically

Reply:
you mean key/value tags, like "host:foo.bar"

Reply:
yes